examples/next/app-router/app/[postSlug]/page.js

```diff
@@ -8,7 +8,7 @@ export default async function Page(props) {
   const postSlug = props.params.postSlug;
 
   // Depending on if isPreview or not use the auth client or regular client
-  let client = getClient();
+  let client = await getClient();
```

Comment:
The `getClient` function has been made `async` for future proofing. Right now there is no need, but there may in the future and I'd rather do the change now before a lot of users are using it.